### ISSUE TITLE
[Accessibility] Cannot trigger "hm-tap" in button using keyboard

### ISSUE THREAD

**Issue by adgoncal** [RyanMullins/angular-hammer#56]
When you use the keyboard "tab" key to focus the button and then press "space" key, hm-tap does not get triggered. Ng-click has the expected behavior.
http://plnkr.co/edit/ooecDAoIrGRLmcbvOsDA?p=preview

Additionally, if I use anchor elements instead, neither hm-tap nor ng-click trigger unless the anchor. In fact, the anchors do not event get focus unles they have an href attribute.
http://plnkr.co/edit/aE99UzMiTJk4khzSjEBK?p=preview

**Comment by adgoncal** [comment]
I am currently adding ng-keydown and checking for "enter" or "space" keyCode as a solution.

http://plnkr.co/edit/MABVJKLLXIQH1xmCpdg3?p=preview

HTML:
```html
<button hm-tap="doSomething($event)"  ng-keydown="doSomething($event)"></button>
<a hm-tap="doSomething($event)"  ng-keydown="doSomething($event)" tabindex="0"></a>
```

JavaScript:
```javascript
function doSomething(event) {
  if(event.type === 'keydown') {
    // quit early if keyCode is not "enter" or "space" key.
    if(!(event.keyCode === 13 || event.keyCode === 32)) {
      return;
    }
  }
  
  // do something ...
}
```
